consensus/src/testing/crash_recovery.rs

```diff
@@ -121,6 +125,26 @@ async fn reconnect_nodes(
     reconnected_nodes
 }
 
+fn verify_backup(buf: &mut &[u8]) -> HashSet<UnitCoord> {
```

Comment:
I know it's a bit redundant with regards to `backup::load_backup`, but nice to have it in test. Please share your thoughts.